packages/logger/tests/unit/Logger.test.ts

```diff
@@ -3595,7 +3608,7 @@ describe('Class: Logger', () => {
       const debugSpy = jest.spyOn(console, 'debug').mockImplementation();
 
       // Act
-      new Logger({ sampleRateValue: 1 });
+      new Logger({ sampleRateValue: 1, logLevel: 'WARN' });
```

Comment:
Sonarcloud reports this line as being useless however it produces a log message as a side effect.
@dreamorosi Is it possible to suppress the warning in sonarcloud?

Reply:
Agreed, I've suppressed it